src/client/components/EmployersQuestions/index.js

```diff
@@ -121,15 +140,23 @@ function EmployerQuestions(props) {
             value={lastDateWorkedDay}
             maxLength={2}
             onChange={(e) => handleDateChange("day", e.target.value)}
+            required
+            pattern="0?[1-9]|1\d|2\d|3[01]"
```

Comment:
This isn't that smart. You could do Feb 30 or Apr 31.

Reply:
I think that's fine!